src/main/java/com/intuit/graphql/authorization/config/AuthzClient.java

```diff
@@ -9,6 +9,7 @@
   private ClientAuthorizationType type;
 
   public enum ClientAuthorizationType {
-    OFFLINE
+    OFFLINE,
+    PRIVATE_AUTH_PLUS
```

Comment:
we should probably refactor this out of this library

Reply:
We can do it as a part of overall library refactor story.